internal/env/env.go

```diff
@@ -66,6 +67,11 @@ func Read(configPath string) (*Config, error) {
 
 		var re = regexp.MustCompile(`[ -]`)
 
+		chainScanner := viper.GetString("CHAIN_SCANNER")
```

Comment:
Can we name it "BLOCK_EXPOLORER"?

Reply:
44b6204b2fa31408e466c5b4bacfc7352981e940